infra/gcp/terraform/k8s-infra-prow-build-trusted/prow-build-trusted/resources/test-pods/test-pods-serviceaccounts.yaml

```diff
@@ -127,3 +127,9 @@ metadata:
     iam.gke.io/gcp-service-account: gcb-builder-cluster-api-gcp@k8s-staging-cluster-api-gcp.iam.gserviceaccount.com
   name: gcb-builder-cluster-api-gcp
   namespace: test-pods
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
```

Comment:
If this will be used for privileged GCS access you'll need to include a workload identity binding like the other `ServiceAccount`s in this file do.

Reply:
Do you have any thoughts here @upodroid?

Reply:
that is right, I forgot about that.

Reply:
Thank you @cjwagner!

@upodroid, I can take a look at this but may need to pair or async on it

Reply:
So we'd need a GCP serviceaccount in the _kubernetes-public_ project, right?
Then the annotation _iam.gke.io/gcp-service-account_ can be added to the serviceaccount.